audioread/__init__.py

```diff
@@ -86,7 +98,10 @@ def available_backends():
     if ffdec.available():
         result.append(ffdec.FFmpegAudioFile)
 
-    return result
+    # Cache the backends we found
+    BACKENDS.extend(result)
```

Comment:
How about `BACKENDS[:] = result` to make it clear that we're always clobbering the entire list here?

Reply:
Sure, I like that.